### PR TITLE
Add pause/resume functionality for Sonus

### PR DESCRIPTION
**Pull request by evancohen**
Usage

``` javascript
sonus = Sonus.init(.......);

Sonus.start(sonus);  // Start
Sonus.pause(sonus);  // Pause
Sonus.resume(sonus); // Resume
```

### CODE REVIEW

**Comment by evancohen** [comment]
@ashishsc yo dawg. This could use some more testing, but you should really look over it. It's pretty complicated...

**Comment by ashishsc** [comment]
will do

File: index.js [hunk 1]
```diff
@@ -106,6 +106,14 @@ Sonus.start = sonus => {
   sonus.mic.pipe(sonus.detector)
 }
 
+Sonus.pause = sonus => {
```
Comment (ashishsc) [inline index.js]: More terse
```javascript Sonus.pause = sonus => sonus.mic.pause()```
```javascript Sonus.resume = sonus => sonus.mic.resume()```


Also, do you want error handling for if its already paused / active?

**Comment by ashishsc** [comment]
Also, why is this failing a build on travis?

**Comment by evancohen** [comment]
Finally 🌵

**Comment by ashishsc** [comment]
lgtm

**Pull request merged**